setup.py

```diff
@@ -59,8 +59,8 @@
         # llvmlite >= 0.32.0 cannot be installed on Python 3.5 and below
         # (building the wheel fails), thus needs to be restricted.
         # See: https://github.com/googleapis/python-bigquery/issues/78
-        "llvmlite <= 0.33.0;python_version>='3.6'",
-        "llvmlite <= 0.31.0;python_version<'3.6'",
+        "llvmlite<=0.34.0;python_version>='3.6'",
+        "llvmlite<=0.34.0;python_version<'3.6'",
```

Comment:
The comment above is correct.
```suggestion
        "llvmlite<=0.31.0;python_version<'3.6'",
```